Api/DTO/Paging.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System.Text.Json.Serialization;
+
+namespace Api.DTO;
+
+public class Paging
+{
+    public const int MinLimit = 1;
+    public const int DefLimit = 10;
+    public const int MaxLimit = 100;
+
+    public int Limit { get; set; }
+    public int Start { get; set; }
```

Comment:
When binding (for example [FromQuery]) object is first constructed using parameterless constructor and then Limit and Start will be assigned to directly. This means that Limit can be assign values outside Min and Max. Add backing field and getter/setter with validation